spec/requests/api/v1/passwords/create_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe 'POST api/v1/users/password', type: :request do
+  let(:user) { create(:user) }
+  let(:email) { user.email }
+
+  let(:params) { { email: email, redirect_url: '' } }
+
+  subject { post user_password_path, params: params }
```

Comment:
maybe add the `as: :json` key to the request here

Reply:
I'm not sure at all but I think it isn't mandatory since she already defined the format in the [routes](https://github.com/flomosq/target-api/blob/e7dc5d0e37dc40537e6d4b11925705f7f9e635e9/config/routes.rb#L8). WDYT? 
@juan-apa